satellite/satellitedb/gracefulexit_test.go

```diff
@@ -0,0 +1,168 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb_test
```

Comment:
The `satellitedbtest` usually test the interface not the particular implementation. For example if we had a completely separate implementation not using "dbx", then these tests wouldn't apply to it. This would mean that it's probably more worthwhile to keep them near the interface rather than the implementation.

Reply:
Moved